app/routes/planet_routes.py

```diff
@@ -0,0 +1,30 @@
+from flask import Blueprint, abort, make_response
+from app.models.planet import planets
+
+# created blueprint
+planets_bp = Blueprint("planets_bp",__name__, url_prefix=("/planets"))
```

Comment:
There's no need to wrap the `url_prefix` value in `()`.

Be sure to include a space after `,`
```py
planets_bp = Blueprint("planets_bp", __name__, url_prefix="/planets")
```